tests/templates/kuttl/smoke/check-s3.py

```diff
@@ -58,23 +58,38 @@ def run_query(connection, query):
     skip_header_line_count = 1
 )
     """)[0][0] is True
-    assert run_query(connection, "SELECT COUNT(*) FROM hive.taxi.taxi_data")[0][0] == 5000
-    rows_written = run_query(connection, "CREATE TABLE IF NOT EXISTS hive.taxi.taxi_data_copy AS SELECT * FROM hive.taxi.taxi_data")[0][0]
-    assert rows_written == 0 or rows_written == 5000
-    assert run_query(connection, "SELECT COUNT(*) FROM hive.taxi.taxi_data_copy")[0][0] == 5000
+    assert run_query(connection, "SELECT COUNT(*) FROM hive.minio.taxi_data")[0][0] == 5000
+    rows_written = run_query(connection, "CREATE TABLE IF NOT EXISTS hive.minio.taxi_data_copy AS SELECT * FROM hive.minio.taxi_data")[0][0]
+    assert rows_written == 5000 or rows_written == 0
```

Comment:
In case the table `hive.minio.taxi_data_copy` already exists nothing will be done and 0 updated rows will be returned.
As for _some_ reasons the table can already exist we don't want to fail the test because of this. Later on we assert that count(*) is 5000 so we are checking for the correct result regardless if the table already existed

Reply:
Ok that works for me.